github/gists.go

```diff
@@ -52,6 +52,30 @@ func (g GistFile) String() string {
 	return Stringify(g)
 }
 
+// GistCommit represents a commit on a gist.
+type GistCommit struct {
+	Version      *string        `json:"version,omitempty"`
+	User         *User          `json:"user,omitempty"`
+	ChangeStatus map[string]int `json:"change_status,omitempty"`
```

Comment:
@shurcooL I see. It didn't occur to me but you're right it seems like the change status has only these exact fields. Thanks.

Would it make sense to move `ChangeStats` type declaration out of `repos_commits.go` to somewhere more shared since it's not specific to repository commits anymore?